ballerina/tests/client_tests.bal

```diff
@@ -59,7 +59,6 @@ function testClientReadTimeout() returns  @tainted error? {
     Error|(readonly & byte[]) res = socketClient->readBytes();
     if (res is (readonly & byte[])) {
```

Comment:
can use `test:assertTrue(res is (readonly & byte[]))`

Reply:
Fixed with 57b72af